fix: ui error when no auth providers configured

Jaskon opened the pull request:
[This issue](https://github.com/umputun/remark42/issues/1456)

<img width="487" alt="image" src="https://github.com/umputun/remark42/assets/18248628/a6093d7e-5a94-4dd5-b9e5-02d5823c6da5">

Please correct me if i'm wrong: view variable (telegram/token/email) is only exist after some action in the dropdown menu. So its not necessary to check here that view is equal to something and checks provided in my PR is enough

Jaskon committed:
fix: ui error when no auth providers configured

paskal commented on frontend/apps/remark42/app/components/auth/auth.messsages.ts:
```diff
@@ -65,4 +65,8 @@ export const messages = defineMessages<string>({
     id: 'auth.signout',
     defaultMessage: 'Sign Out',
   },
+  noProviders: {
```
I think you need to add a translation string and regenerate translations in case you're adding new text like that.

paskal commented:
@Jaskon can you please finalise that PR so that we could merge it?

paskal committed:
fix: correct no-providers message grammar, translate it, and cover both branches

Reword "May be" to "Maybe" in the auth.no-providers message and run
translation:generate to register the key in every locale dictionary, then
replace the English placeholders with proper translations for each locale.
Add a test asserting the error is hidden when providers are configured.

paskal commented:
Rebased onto current master to clear the conflict (the messages file was renamed from `auth.messsages.ts` to `auth.messages.ts` after this PR was opened). Added a small follow-up commit: corrected the message grammar ("May be" → "Maybe") and ran `translation:generate` to propagate the new `auth.no-providers` key into all locale catalogs. Verified locally — eslint, type-check, translations check, and `jest` (including the new test) all pass.

paskal commented:
Updated the follow-up commit: replaced the English placeholders with proper translations for all 23 non-English locales (the new `auth.no-providers` key), and added a test asserting the error is hidden when providers *are* configured (covers both branches of the new conditional). Local checks green: translations check, type-check, eslint, jest (30 tests).

Note on `codecov/project`: codecov reports "all modified and coverable lines are covered" (`codecov/patch` passes). The `-1.72%` on `codecov/project` is a stale-base artifact — the report is 8 commits behind head and shows Files −8 / Lines −148 / Partials +108 across unrelated code, not anything introduced here.
